ts/Data/DataTable.ts

```diff
@@ -1417,6 +1446,67 @@ class DataTable implements DataEvent.Emitter {
         });
     }
 
+    /**
+     * Sets the row key column. This column is invisible and the cells
+     * serve as identifiers to the rows they are contained in. Accessing
+     * rows by keys instead of indexes is necessary in cases where rows
+     * are rearranged by a DataModifier (e.g. SortModifier or RangeModifier).
+     *
+     * @function Highcharts.DataTable#setRowKeysColumn
+     *
+     * @param {number} nRows
+     * Number of rows to add to the column.
+     *
+     */
+    public setRowKeysColumn(nRows: number): void {
+        const id = this.rowKeysId;
+        if (id) {
+            this.columns[id] = [];
+            const keysArray = this.columns[id];
+
+            for (let i = 0; i < nRows; i++) {
+                keysArray.push(id + '_' + String(i));
+            }
+        }
+    }
+
+    /**
+     * Get the row key column.
+     *
+     * @function Highcharts.DataTable#getRowKeysColumn
+     *     *
+     * @return {DataTable.Column|undefined}
+     * Returns row keys if rowKeysId is defined, else undefined.
+     */
+    public getRowKeysColumn(): DataTable.Column|undefined {
+        const id = this.rowKeysId;
+        if (id) {
+            return this.columns[id];
+        }
```

Comment:
In this case we do not return anything. 
IMO: `return this.columns[id];` will return value or undefined, so the IF is unecessary.

Reply:
I use the if because it gets rid of a lint error: 
`Type 'undefined' cannot be used as an index type.ts(2538)`